src/main/kotlin/tanvd/grazi/ide/ui/GraziListToolbarDecorator.kt

```diff
@@ -0,0 +1,35 @@
+package tanvd.grazi.ide.ui
+
+import com.intellij.ui.CommonActionsPanel
+import com.intellij.ui.RowsDnDSupport
+import com.intellij.ui.ToolbarDecorator
+import com.intellij.util.ui.EditableModel
+import tanvd.grazi.language.Lang
+import javax.swing.JList
+
+class GraziListToolbarDecorator(val list: JList<Any>) : ToolbarDecorator() {
```

Comment:
If it is used only inside GraziAddDeleteListPanel -- why no to make it nested class?

Reply:
Done